setup.py

```diff
@@ -58,25 +58,25 @@ def is_requirement(line):
 
 
 setup(
-    name='django-babel-underscore',
+    name='enmerkar-underscore',
     version=version,
     description='Implements a underscore extractor for django-babel.',
     long_description=read('README.rst') + u'\n\n' + read('HISTORY.rst'),
     author='Christopher Grebs',
     author_email='cg@webshox.org',
-    url='https://github.com/EnTeQuAk/django-babel-underscore',
+    url='https://github.com/Zegocover/enmerkar',
```

Comment:
This should be the URL of this repo.  And the author and author_email should also be updated to the standard edX values.